cdflow_commands/ecs_monitor.py

```diff
@@ -73,6 +74,8 @@ def __init__(self, cluster, environment, component, version, boto_session):
         self._boto_session = boto_session
         self._done = False
         self._seen_ecs_service_events = set()
+        self._new_service_deployment = None
+        self._new_service_deployment_grace_period_count = 0
```

Comment:
I find the name of this and `NEW_SERVICE_DEPLOYMENT_GRACE_PERIOD_LIMIT` a bit confusing. I'd suggest something like `new_deployment_buffer_iterations` or `new_deployment_grace_iterations`or something like that.